Source/ACE/Entity/Player.cs

```diff
@@ -274,10 +275,10 @@ public void Load(AceCharacter character)
             IsAlive = true;
             IsOnline = true;
 
-            PhysicsData.MTableResourceId = Character.MotionTableId;
-            PhysicsData.Stable = Character.SoundTableId;
-            PhysicsData.Petable = Character.PhysicsTableId;
-            PhysicsData.CSetup = Character.ModelTableId;
+            MotionTableId = Character.MotionTableId;
```

Comment:
They should be fine to remove.   I did not remove them yet because I have done little to no work on player and did not really have time to test.   I would really hate to hold this up for that.   How about a big todo and when I have more time, I will clean it up?